README.md

```diff
@@ -264,39 +243,40 @@ For `nvim-treesitter` to work, we need to use query files such as those you can
 We are looking for maintainers to write query files for their languages.
 
 List of currently supported languages:
-- [x] lua  (maintained by @vigoux)
-- [x] ruby (maintained by @TravonteD)
+
+- [x] bash (maintained by @TravonteD)
 - [x] c (maintained by @vigoux)
-- [x] go (maintained by @theHamsta, @WinWisely268)
 - [x] cpp (maintained by @theHamsta, extends C queries)
-- [x] rust (partial support, maintained by @vigoux)
-- [x] python (maintained by @theHamsta)
-- [x] javascript (maintained by @steelsojka)
-- [x] typescript (maintained by @steelsojka)
-- [ ] tsx
-- [x] json (maintained by @steelsojka)
-- [x] html (maintained by @TravonteD)
 - [x] csharp (maintained by @svermeulen)
-- [ ] swift
-- [x] java
-- [ ] ocaml
 - [x] css (maintained by @TravonteD)
+- [x] dart (maintained by @Akin909)
+- [ ] elm
+- [x] go (maintained by @theHamsta, @WinWisely268)
+- [ ] haskell
+- [x] html (maintained by @TravonteD)
+- [x] java
+- [x] javascript (maintained by @steelsojka)
+- [ ] jsdoc
+- [x] json (maintained by @steelsojka)
 - [ ] julia
+- [x] lua  (maintained by @vigoux)
+- [ ] markdown
+- [ ] nix
+- [ ] ocaml
 - [ ] php
-- [x] bash (maintained by @TravonteD)
+- [x] python (maintained by @theHamsta)
+- [x] regex (maintained by @theHamsta)
+- [x] rst (maintained by @stsewd)
+- [x] ruby (maintained by @TravonteD)
+- [x] rust (partial support, maintained by @vigoux)
 - [ ] scala
-- [ ] haskell
+- [ ] swift
 - [ ] toml
+- [x] tree-sitter query language (maintained by @steelsojka)
+- [ ] tsx
+- [x] typescript (maintained by @steelsojka)
 - [ ] vue
-- [ ] elm
 - [ ] yaml
-- [ ] nix
-- [ ] markdown
-- [x] regex (maintained by @theHamsta)
-- [ ] jsdoc
-- [x] dart (maintained by @Akin909)
-- [x] rst (maintained by @stsewd)
-- [x] tree-sitter query language (maintained by @steelsojka)
 
```

Comment:
The rendered version looks weird here, but I don't know if that comes from the previewer or not.

Reply:
Was the previewer's fault